src/Sulu/Bundle/AdminBundle/Resources/js/stores/UserStore/UserStore.js

```diff
@@ -26,10 +29,24 @@ class UserStore {
         this.resetSuccess = false;
     }
 
-    @computed get locale() {
+    @computed get systemLocale() {
         return this.user ? this.user.locale : Config.fallbackLocale;
     }
 
+    @computed get contentLocale() {
+        const userLocale = this.getPersistentSetting(CONTENT_LOCALE_SETTING_KEY);
+
+        if (!userLocale) {
+            localizationStore.loadLocalizations().then((localizations) => {
+                const defaultLocalizations = localizations.filter((localization) => localization.default);
+                const fallbackLocalization = defaultLocalizations.length ? defaultLocalizations[0] : localizations[0];
```

Comment:
Why don't you just return this value? Would be enough for now, and we are not sure yet if the locale will be read from the user settings afterwards.